core/store/migrations/0053_add_fmv2_drumbeat_random_delay.go

```diff
@@ -0,0 +1,25 @@
+package migrations
+
+import (
+	"gorm.io/gorm"
+)
+
+const up53 = `
+    ALTER TABLE flux_monitor_specs ADD COLUMN drumbeat_random_delay bigint;
```

Comment:
NOT NULL default 0 ?

Reply:
I saw in some previous update that another column could be null, so was inspired by that :)